examples/braket_features/Noise_models/Noise_models_on_Rigetti_AspenM1.ipynb

```diff
@@ -0,0 +1,1064 @@
+oealq5UoOptZQ+0/nLj1gC9PuyS9508pdefPK1WvQi8mzu/fs5clodzx506Z1CztTdnKA+QsWMxX6vYXX8qULIRfTHufLpZ5iU00pYNpDh4+CiZMiTfLIBC1n353LzPD07NVXvB7QrzfUq1tbey5VpoLuiSIRmDmQOJUqVSqtbFWrfANvvfWW/Fq4ZSIV/g5UEzwiIDkIgQAhENX7c3f7QLP6c9S2M4Wd0Bw+crTTGsbNrurVqkD9urXsDjWZ1Z8/fvwYRo4aB3PmzbfLA/ZjFcqXg5Ytmtr584eLFy9pWlv4s7x4yP08vaOWwExZc4poqsaZmbPmwrIV3zoleSFWHZkpuErflLdTqe9qnHaXsNS5a09YvGSplj9cV5o+daLIq+yQNxT05Hdc5B4waKiu1iLcTO7MtB5WrvQNMyvaT5hg09vkxN9h23YdhWYpHEfwBK9qZtXdMfv8+QtMDulptynPy4UaRZqxhXY0HXfs+HHN2yzCkjN5x5f2wfO9a/de6Mg0ozgjWeO6DJoOdDZO+7omiJqc5s5bqGlV4XlS7+3YGiCuUcaMGVN9ZfgsyyPutL/GTVvCDxs2ijQ3/rAW0n+WDvAEdNr0mYS/ngN/W8kZOSNF8uSa2UjZZLEaXiZSGbUTNR49WwuBcJMXLp0/bUgYVDVj6PXtuPnYsUt3w3kCxqtbp5a2l8BrHEmDAwcPZdqEVnIvccdNxx5s72HR4qUwduQ23hYAACzsSURBVHzkmKNHWMIIOIaOHD1e1wQ3tkXclENNS1269dTSVzVFiY8yR7D7Tjkv7o5hchxnbnkuijKDM1PcKmHJmQl0kgcB/CEPOqs/8rcWAlYdJ1TNLhPGjYby5cq4Db48r8BI6vhy6/Yd6N2nv92anJw4mlTszfrrjz5KIHuDu3M2V/MeOVEkc8hac2XikxxOdZsho6tp8md5/9QdwhLG82b8NUNG37BxM3Tr3ksQgXkZ+B3l5/Lly2p7/smTJeXe4o5jbbcevQSvAF9gnMzMahRqOf70009EWFeOps1bCwJVtSqVYcTwwbpR0Oz44KHDtXdIOorYs0M33Lr1G9gaxRiHA/c4dnZo34ZpBy7PiH2ZBbFPj7CECfuyfu1qruvub91bOUKWGbAsZqxfYzp0BQaBkCAs8aIiE/YSUwd++syPTKCfYNfoixcryuyD1tbMu7322ms8itM7nnY4++M51jivwV+sE0+RPJm2CfH+++87jePOC1ykQluhz54909T+GTFX3Ukv3MJYVagJt3qg8hACZiLA2zWmaVVig5l4UFqEQLggwNs2tWv3avTevfuaqZ0nfz6BFCmTa6eX3JFJ1dRRlsTTTdeYSbn79x/C//73AVM/HJeZ8vkIEiVMqAa3e8aFiV8fPYL48eLrEuXtAhs8YDp3mJmB6NGjs3J8ZLdBrBdNnvDhRPdgxC6XcfTSkf1wsSTD51nFRFnvtLAcPlTc6sk9q+Q7VPCjfPgHAerPPcPVrP4cNRr89NNP2poDLmKjSbh48eKy//G09YfXX3/dacbM6s95Hv5laymfJE4Eb775ptNv4osLFy5C4aIlRZiIvTu1eMLDS0fDxs2FSTQ8JLZhvc1EJ08ST3PeYXihRrzHj39nOMXRTKMiycoZ8YTHDaU7mndAM65YDlwwxvwnSBAfPJUJcAPjn3+ear8Zo7jujNm4/nWN5QvXv168eA5pUqfW8mb0G/Q3pvy3iXnytH3wvCHGly5fZsSln+G9996FFClSaCZd9Q4L8jj8bsaaIGo0wfpGU4/4//33Y0Fi1s5wfTFWrFj8U5a/q6fFF8ybxcwG57d8uagAjgiQvOCICff5/fff4dq168yk23XNtAuah4nH5mioxU02t8jD8zv2T7hP8Ouvv0JKNh5g/+DNmIbfv87miNjXYD+TLGlSr+d7we47OTZ4d2cMk8MH0k3yoPnyYCDrj77lHwRonDDGFeXLn3++p63nvXz5immV/RASsT4bCStGF5eL3Z2zGaXl6zueF19kdF/zIMf3Zvz1VUZ/9eqVNt7iuH/txg0tO/HZPB7XZnEcjxEjhpxFXTfO5VAGiMPkBTy8EioX/kZx3o0HON544w1IzQ5l4rqzN5cv69fuznVd5ctTOcIf69eu8kjvzUMgpAhLcrHQ7iOeTFNPKqCK7dy5vtQWYFKnSqnZYfT0RJP8HXKbiwAJNebiSakRAqGAAG/XmBciNoRCjVAeCAFzEOBtm9q1OXiGcyolSpVjBwrOaEV097SUO3jI5nVwgedAxG6IHdu3wwXufNeXMKo6cFVThS9pU1xCwFsEqD/3FrmoF2/Q4GEwZdoMreD58ubRzFmZgQJqt67EzNrwa8WyRdphM/5Md0KAEAhdBNq06wQrV63WMogkvN07tnpNlAjdUlLOEAGSF+h3QAgQAoiAv+RBQtf6CNA4Yf06pBIQAlEZAX+tX0dlTANZ9pAlLHEQtmzdrqndvnr1Kveyu6vqxu1e0kPAESChJuCQ0wcJAb8jwNs1foiIDX6Hmz5ACAQMAd62qV0HDHJLfgjVHdeoVVfk/fCBvQ5qrsVLDx14QCE3M9GMp6PwCnVtRXiyJ2fuArb8Nm2smZr2sNgUnBAwHQHqz02HNCwTxL4W1fXza+rkCVC6VAn+6PO9QaNmsGnzFi0dNHG6bs1Kj7UO+ZwJSoAQIAQ8QkA1maVnxtCjBClwSCNA8kJIVw9ljhAICAL+lgcDUgj6iN8QoHHCb9BSwoQAIeBnBPy5fu3nrFPy/yEQ8oQlzCeqqftuzVqYNXueON3Na5AISxyJ0LiTUBMa9UC5IATMRIC3a0yTiA1mIktpEQLBRYC3bWrXwa2HUPs6qmO+x0ywvBktGuzZG8FOXw4VZtsqV6oIo0cOMzXLq1avgdZtO4g0D+7frZnbEx4h5JDtxqNGqEMH9oSVSZgQgpqy4iEC1J97CFgUCf7ixQu4yEyXvf/++3D8xEmYPHU6nDx5Sis9mvfcz8zBuWNay1240H
```

Comment:
Can you add the respective images in the folder and source them via HTML as per the contribution guidelines: https://github.com/aws/amazon-braket-examples/blob/main/CONTRIBUTING.md

Our images are not rendered properly sometimes we added them as attachments: https://github.com/aws/amazon-braket-examples/pull/148